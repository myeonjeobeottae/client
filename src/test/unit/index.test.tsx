import LandingPage from '@pages/index';
import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';

describe(`<LandingPage />`, () => {
	it('랜딩페이지 링크 버튼을 누르면 해당 페이지로 이동한다.', () => {
		render(<LandingPage />);
		const chatButton = screen.getByRole('link', { name: 'chat' });
		userEvent.click(chatButton);

		expect(chatButton).toHaveAttribute('href', '/chat');

		const voiceButton = screen.getByRole('link', { name: 'voice' });
		userEvent.click(chatButton);

		expect(voiceButton).toHaveAttribute('href', '/voice');
	});
});
