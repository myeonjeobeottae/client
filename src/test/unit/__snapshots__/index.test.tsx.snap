// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Create Next App 1`] = `
<div>
  <div>
    Home
  </div>
</div>
`;
